assets/js/cookieBanner.js

```diff
@@ -21,10 +21,21 @@
   }
 
   function hideCookiesConfirmation(banner) {
-    banner.querySelectorAll('#cookie-banner-accepted, #cookie-banner-rejected')
-      .forEach(message => message.hidden = true)
+    banner
+      .querySelectorAll('#cookie-banner-accepted, #cookie-banner-rejected')
+      .forEach(message => (message.hidden = true))
 
-      banner.hidden = true
+    banner.hidden = true
+  }
+
+  function removeAnalyticsCookies() {
+    const expires = new Date(0).toUTCString()
+    const gaID = document
+      .querySelector('[data-google-analytics-id]')
+      ?.getAttribute('data-google-analytics-id')
+      .replace('G-', '')
+    document.cookie = `_ga=; expires=${expires}; path=/;`
+    document.cookie = `_ga_${gaID}=; expires=${expires}; path=/;`
```

Comment:
These are the actual intended changes to this file, to remove cookies (the rest is just `prettier`)